docs/scenarios-workflow.md

```diff
@@ -0,0 +1,90 @@
+
+# Scenario Tests Guide
```

Comment:
Should we also list any dependencies that need to be installed on the users machine as well? I know that we need python, but is there anything else that we need on either Linux or Windows?

Reply:
Good point! We can add a prerequisites section. I think other dependencies needed will be netcoreapp3.0 (as is the default version for building startup tool).